cpp/betweenness_centrality_module/algorithm_online/betweenness_centrality_online.cpp

```diff
@@ -0,0 +1,316 @@
+  std::unordered_map<std::uint64_t, double> normalized_bc_scores;
+  for (const auto [node_id, bc_score] : node_bc_scores) {
+    normalized_bc_scores[node_id] = bc_score * normalization_factor;
+  }
+
+  return normalized_bc_scores;
+}
+
+void OnlineBC::BrandesWrapper(const mg_graph::GraphView<> &graph, const std::uint64_t threads) {
+  this->node_bc_scores.clear();
+
+  const auto bc_scores = betweenness_centrality_alg::BetweennessCentrality(graph, this->directed, false, threads);
+  for (std::uint64_t node_id = 0; node_id < graph.Nodes().size(); ++node_id) {
+    this->node_bc_scores[graph.GetMemgraphNodeId(node_id)] = bc_scores[node_id];
+  }
+}
+
+bcc_data OnlineBC::IsolateAffectedBCC(const mg_graph::GraphView<> &graph,
+                                      const std::pair<std::uint64_t, std::uint64_t> updated_edge) const {
+  std::unordered_set<std::uint64_t> articulation_points_by_bcc;
+  std::vector<std::unordered_set<std::uint64_t>> nodes_by_bcc;
+  const auto edges_by_bcc = bcc_algorithm::GetBiconnectedComponents(graph, articulation_points_by_bcc, nodes_by_bcc);
+
+  std::unordered_set<std::uint64_t> affected_bcc_nodes;
+  std::unordered_set<std::uint64_t> affected_bcc_articulation_points;
+
+  for (std::size_t i = 0; i < edges_by_bcc.size(); i++) {
+    if (std::any_of(edges_by_bcc[i].begin(), edges_by_bcc[i].end(), [updated_edge](auto &edge) {
+          return edge.from == updated_edge.first && edge.to == updated_edge.second;  // if edge in BCC
+        })) {
+      for (const auto node : nodes_by_bcc[i]) {
+        affected_bcc_nodes.insert(node);
+      }
+      for (const auto node : articulation_points_by_bcc) {
+        if (affected_bcc_nodes.count(node)) affected_bcc_articulation_points.insert(node);
+      }
+    }
+  }
+
+  return {affected_bcc_nodes, affected_bcc_articulation_points};
+}
+
+std::unordered_map<std::uint64_t, int> OnlineBC::SSSPLengths(
+    const mg_graph::GraphView<> &graph, const std::uint64_t source_node_id,
+    const std::unordered_set<std::uint64_t> &affected_bcc_nodes) const {
+  std::unordered_map<std::uint64_t, int> distances;
+  distances[source_node_id] = 0;
+
+  std::queue<std::uint64_t> queue({source_node_id});
+  while (!queue.empty()) {
+    const auto current_id = queue.front();
+    queue.pop();
+
+    for (const auto neighbor_id : graph.GetNeighboursMemgraphNodeIds(graph.GetInnerNodeId(current_id))) {
+      if (!affected_bcc_nodes.count(neighbor_id)) continue;
+
+      if (!distances.count(neighbor_id)) {  // if unvisited
+        queue.push(neighbor_id);
+        distances[neighbor_id] = distances[current_id] + 1;
+      }
+    }
+  }
+
+  return distances;
+}
+
+std::unordered_map<std::uint64_t, int> OnlineBC::PeripheralSubgraphsOrder(
+    const mg_graph::GraphView<> &graph, std::unordered_set<std::uint64_t> affected_bcc_articulation_points,
+    std::unordered_set<std::uint64_t> affected_bcc_nodes) const {
```

Comment:
This throws a syntax error because `affected_bcc_articulation_points` and `affected_bcc_nodes` are already rvalue references since they're made by structured binding.